apollo-ios/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## v1.15.0
+
+### New
+- **Add ability to disable fragment field merging ([#431](https://github.com/apollographql/apollo-ios-dev/pull/431)):** Added `ApolloCodegenConfiguration` option to allow for disabling fragment field merging on generated models.
```

Comment:
Can we link to the notes on this feature from the preview release here: https://github.com/apollographql/apollo-ios/releases/tag/preview-field-merging.1

There is a bug fix that was inadvertently fixed by this work as well, but it does make for a minor breakage of existing code in some situations (not a major version since this is a bug fix). In that preview release, I explain it under the heading: "Bug Fix: Generated Selections Sets in Inclusion Condition Scope".

Can we copy that text into an item in the change log under the "Breaking" header, just so people are aware.